Interfaces/API/Dirac.py

```diff
@@ -963,16 +977,12 @@ def runLocal( self, jobJDL, jobXML, baseDir, disableCopies = False, disableCallb
       errorFile = open( errorFileName, 'w' )
       print >> errorFile, stderr
       errorFile.close()
+      sandbox = None
```

Comment:
It is maybe cleaner and safer to define 'sandbox' before the test case, so in case there is an 'elif' added once, you're sure to have the variable defined for your test line 985